### DISCUSSION TITLE
Prusa Slicer Configuration on SuperSlicer

### DISCUSSION THREAD

**Discussion by rainer1146** [supermerill/SuperSlicer#2163] [Q&A]
Hi
I have been using the Prusa Slicer so far and have already accumulated a few profiles.
Is it recommended to use these profiles with SuperSlicer?
Or can there be strange attitudes?
regards Rainer

**Comment by supermerill** [comment]
you can have some strange settings, if you are using:
 -  toolchange_gcode add `T[next_extruder]`
 -  xy_size_compensation:set also xy_inner_size_compensation
 - if infill_anchor_max = 0then set infill_connection to notconnected
 - first_layer_speed, also set first_layer_min_speed
It's if you want to copy-paste your profiles directly into your configuration directory.
If you're using the GUI to transfer some, you can use the "import from prusa" so it does these things itself.

**Comment by rainer1146** [comment]
Alright, thank you very much.
Then the easiest way will be to use the GUI.
I had not yet discovered this point because I always tried using 'Configuration'.
